src/ui/widgets/Display/display.tsx

```diff
@@ -54,11 +55,15 @@ export const DisplayComponent = (
   style["position"] = "relative";
   style["overflow"] = props.overflow;
   style["height"] = "100%";
-  if (props.autoZoomToFit) {
-    const internalScale =
-      typeof props.scaling === "undefined" ? "1" : props.scaling;
+  const internalScale =
```

Comment:
Some comments here explaining this logic would be good. I know you have written some further down but if you come to this  in isolation it is hard to understand

Reply:
I have now added a comment to clarify this how this value is determined.